diff --git a/bindings/dotnet/CMakeLists.txt b/bindings/dotnet/CMakeLists.txt
index abcdef12..12345678 100644
--- a/bindings/dotnet/CMakeLists.txt
+++ b/bindings/dotnet/CMakeLists.txt
@@ -44,7 +44,7 @@ set(dotnet_config --configuration $<CONFIG>_Build)
 # suppress warnings:
 # > CS1591: Missing XML comment for publicly visible type or member 'xyz'.
 # > NU1801: The local source 'path' doesn't exist.
-set(msbuild_parameters -p:NoWarn="CS1591,NU1801" -p:CMAKE_RUNTIME_OUTPUT_DIRECTORY=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} -p:OPENDAQ_PACKAGE_VERSION=${OPENDAQ_PACKAGE_VERSION})
+set(msbuild_parameters -p:NoWarn="CS1591,NU1801" -p:Platform=${platform} -p:CMAKE_RUNTIME_OUTPUT_DIRECTORY=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} -p:OPENDAQ_PACKAGE_VERSION=${OPENDAQ_PACKAGE_VERSION})
 set(dotnet_cmd_args ${projectFileName} ${dotnet_config} --verbosity minimal ${nuget_config} ${msbuild_parameters})
 set(dotnet_test_args ${projectFileName} --no-build ${dotnet_config} --logger """junit\\\;LogFilePath=${test_path}/test_${targetName}.xml""")
 #set(dotnet_pack_args ${projectFileName} --no-build ${dotnet_config} ${msbuild_parameters})
