diff --git a/examples/modules/basic_csv_recorder_module/include/basic_csv_recorder_module/multi_csv_writer.h b/examples/modules/basic_csv_recorder_module/include/basic_csv_recorder_module/multi_csv_writer.h
--- a/examples/modules/basic_csv_recorder_module/include/basic_csv_recorder_module/multi_csv_writer.h
+++ b/examples/modules/basic_csv_recorder_module/include/basic_csv_recorder_module/multi_csv_writer.h
@@ -17,6 +17,7 @@
 #pragma once

 #include <fstream>
+#include <condition_variable>
 #include <queue>
 #include <string>
 #include <thread>
