diff --git a/bindings/dotnet/openDAQ.Net/openDAQ.Net/openDAQ.Net.csproj b/bindings/dotnet/openDAQ.Net/openDAQ.Net/openDAQ.Net.csproj
index abcdef12..12345678 100644
--- a/bindings/dotnet/openDAQ.Net/openDAQ.Net/openDAQ.Net.csproj
+++ b/bindings/dotnet/openDAQ.Net/openDAQ.Net/openDAQ.Net.csproj
@@ -17,7 +17,7 @@
     <OPENDAQ_PACKAGE_VERSION Condition="'$(OPENDAQ_PACKAGE_VERSION)' == ''">0.0.0</OPENDAQ_PACKAGE_VERSION>
     <_BuildNumber>23</_BuildNumber>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <PlatformTarget>x64</PlatformTarget>
+    <PlatformTarget>$(Platform)</PlatformTarget>
     <Title>openDAQ SDK .Net-Bindings</Title>
     <Authors>openDAQ</Authors>
     <Description>.Net bindings to the openDAQ SDK (open data acquisition software development kit)</Description>
